test-requirements.txt

```diff
@@ -1,7 +1,5 @@
-black==22.3.0 ; python_version > '3.5'
+black==23.3.0 ; python_version >= '3.9'
 flake8
-git+https://github.com/ansible-community/pytest-ansible-units.git
 mock
```

Comment:
I think we can remove mock and coverage too. Since validated content won't have unit tests.